coderd/database/dbauthz/dbauthz.go

```diff
@@ -3554,6 +3554,17 @@ func (q *querier) ListWorkspaceAgentPortShares(ctx context.Context, workspaceID
 	return q.db.ListWorkspaceAgentPortShares(ctx, workspaceID)
 }
 
+func (q *querier) MarkAllInboxNotificationsAsRead(ctx context.Context, arg database.MarkAllInboxNotificationsAsReadParams) error {
+	resource := rbac.ResourceInboxNotification
+	resource.Owner = arg.UserID.String()
```

Comment:
nit: I think this should be

```suggestion
	resource := rbac.ResourceInboxNotification.WithOwner(arg.UserID.String())
```

Reply:
✅